### PR TITLE
7.x-4.x-BASIC-1259: Changed upcoming_events context

### PR DESCRIPTION
**Pull request by cynmij**

### CODE REVIEW

**Comment by cynmij** [comment]
This fixes the order of main menu and exposed filter in first sidebar

File: modules/stanford_jsa_layouts/stanford_jsa_layouts.context.inc [hunk 1]
```diff
@@ -11,7 +11,7 @@ function stanford_jsa_layouts_context_default_contexts() {
   $export = array();
 
   $context = new stdClass();
-  $context->disabled = TRUE; /* Edit this to true to make a default context disabled initially */
+  $context->disabled = FALSE; /* Edit this to true to make a default context disabled initially */
```
Comment (sherakama) [inline modules/stanford_jsa_layouts/stanford_jsa_layouts.context.inc:14]: Are you sure you want to enable this?

File: modules/stanford_jsa_layouts/stanford_jsa_layouts.context.inc [hunk 2]
```diff
@@ -364,13 +364,7 @@ function stanford_jsa_layouts_context_default_contexts() {
           'module' => 'views',
           'delta' => '-exp-stanford_events_views-page',
           'region' => 'sidebar_first',
-          'weight' => '3',
-        ),
-        'menu_block-stanford_jsa_layouts-1' => array(
```
Comment (sherakama) [inline modules/stanford_jsa_layouts/stanford_jsa_layouts.context.inc:369]: What menu block is missing? Are you sure you want to remove this one?

File: modules/stanford_jsa_layouts/stanford_jsa_layouts.info [hunk 3]
```diff
@@ -11,6 +11,7 @@ dependencies[] = features
 dependencies[] = menu
 dependencies[] = menu_block
 dependencies[] = search
+dependencies[] = stanford_event_series
```
Comment (sherakama) [inline modules/stanford_jsa_layouts/stanford_jsa_layouts.info:14]: Where did this dependency come from?

File: modules/stanford_jsa_layouts/stanford_jsa_layouts.context.inc (hunk 2 above)
Comment (cynmij) [inline modules/stanford_jsa_layouts/stanford_jsa_layouts.context.inc:369]: Yes.  Menu block is placed in all_pages_but_homepage_jsa Context

**Comment by cynmij** [comment]
trying this again on branch BASIC-1259

**Pull request closed without merging**